frontend/src/components/MapView/Layers/AnalysisLayer/index.tsx

```diff
@@ -77,7 +79,22 @@ const onClick = (analysisData: AnalysisResult | undefined) => ({
     );
     dispatch(
       addPopupData({
-        [analysisData.getStatTitle(t)]: {
+        [t('Analysis layer')]: {
+          data: analysisData.getLayerTitle(t),
+          coordinates,
+        },
+        ...(analysisData.analysisDate
+          ? {
+              [t('Date analyzed')]: {
+                data: getFormattedDate(
+                  analysisData.analysisDate,
+                  DateFormat.DayFirst,
```

Comment:
I'll let @wadhwamatic decide but I'm tempted to use localDate here instead of DayFirst to display what the user is used to?

Reply:
I also vote for localDate

Reply:
Updated ✅ 
<img width="289" alt="Screenshot 2024-06-19 at 8 56 56 AM" src="https://github.com/WFP-VAM/prism-app/assets/8203830/17eb7023-3619-4f00-b05b-310b394ae6ab">